src/main/java/ToDo.java

```diff
@@ -0,0 +1,9 @@
+public class ToDo extends Task {
```

Comment:
Perhaps name this class to "Todo"? so that it is more similar to the related subclasses (i.e. Event, Deadline)